tests/phpunit/elementor/core/experiments/test-manager.php

```diff
@@ -82,6 +83,29 @@ public function test_add_feature__ensure_wrap_core_dependency() {
 		$this->assertEquals( 'core_feature', $depended_feature_dependency->get_name() );
 	}
 
+	public function test_feature_can_be_added_as_hidden() {
+		// Act
+		$args = [ 'hidden' => true ];
+		$this->add_test_feature( $args );
+
+		// Assert
+		$feature = $this->experiments->get_features( 'test_feature' );
+		self::assertNotEmpty( $feature );
+		self::assertTrue( $feature['hidden'] );
+	}
+
+	public function test_features_are_added_as_not_hidden_by_default() {
+		// Act
+		$this->add_test_feature();
+
+		// Assert
+		$features = $this->experiments->get_features();
```

Comment:
Is the feature list empty before you add the test feature? our CI tests activate a bunch of features by default during the run, so it might not be empty regardless of whether you add a test feature here or not